### DISCUSSION TITLE
restructuredtext support

### DISCUSSION THREAD

**Discussion by mnemonico** [orhun/git-cliff#747] [General]
i started using the lib on some of my projects, it's really useful thanks a lot for the work, now i've been thinking since i have all my project using rst with sphinx doc building, could it be a way to have support on restructuredtext template to generate .rst changelog file ? is it something you think of ?

**Comment by orhun** [comment]
Hello! 👋🏼 You can customize the changelog format to generate a .rst file. `git-cliff` uses [Tera](https://keats.github.io/tera/docs/) templates which gives you the flexibility to format the changelog output as needed.

For example:

```toml
[changelog]
body = """
Release Notes
=============
{% for group, commits in commits | group_by(attribute="group") %}
{{ group }}
{% for commit in commits %}
- {{ commit.message | capitalize }}
{% endfor %}
{% endfor %}
"""
```

Check out [documentation](https://git-cliff.org/docs/) and [examples](https://git-cliff.org/docs/templating/examples) for more information.
